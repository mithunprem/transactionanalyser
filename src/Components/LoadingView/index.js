import React, { Fragment } from 'react';
import { Spinner } from 'reactstrap';

const LoadingView = () => {
  return (
    <Fragment>
      <Spinner type="grow" />
      <h5 className="mt-3">Loading transactionsâ€¦</h5>
    </Fragment>
  );
}

export default LoadingView;
