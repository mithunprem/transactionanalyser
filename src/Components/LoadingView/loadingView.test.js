import React from 'react';
import { create } from 'react-test-renderer';
import LoadingView from '../../Components/LoadingView';

describe("LoadingView", () => {
  test("should render without throwing an error", () => {
    const loadingView = create(<LoadingView />).toJSON()
    expect(loadingView).toMatchSnapshot()
  });
});
