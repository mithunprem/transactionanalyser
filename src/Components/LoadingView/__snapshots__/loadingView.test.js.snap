// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingView should render without throwing an error 1`] = `
Array [
  <div
    className="spinner-grow"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>,
  <h5
    className="mt-3"
  >
    Loading transactionsâ€¦
  </h5>,
]
`;
